lib/utils/matrix_sdk_extensions/event_extension.dart

```diff
@@ -158,4 +165,66 @@ extension LocalizedBody on Event {
       room.isDirectChat ||
       !isSameSenderWith(nextEvent) ||
       type == EventTypes.Encrypted;
+
+  bool get isPinned => room.pinnedEventIds.contains(eventId);
+
+  Future<bool> unpin() async {
+    if (!isPinned) {
+      return true;
+    }
+    final pinnedEventIds = room.pinnedEventIds;
+    pinnedEventIds.remove(eventId);
+    String? message;
+    try {
+      message = await room.setPinnedEvents(pinnedEventIds);
```

Comment:
What is `message`?

Reply:
The name is confused, change it to `responseMessage`
<img width="704" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/43041967/d3cdbe52-c444-464b-b952-23bcb78b9b02">